gordon_gcp/schemas/examples/audit-log.first-operation.json

```diff
@@ -0,0 +1,54 @@
+{
+  "insertId": "123456789101112",
+  "logName": "projects/some-project/logs/cloudaudit.googleapis.com%2Factivity",
+  "operation": {
```

Comment:
This `"operation"` key could be interesting; it could potentially help us avoid unnecessary polling Compute Engine if we know that if there's ever an `"operation"` key, that there will be at least 2 messages, one with a `"first"` key set to true, and another with `"last"` key set to true. If that's the case, we can shorten the polling of GCE, as well as only act if the operation was completed. The [docs](https://cloud.google.com/logging/docs/audit/api/#large_or_long-running_audit_log_entries) are a little unclear.

Reply:
I'm not sure but I *think* the one with no operations gets issued at the same time as the "last".  I was only looking at the sample data (which of course I didn't think to save anywhere to I can find it now), so I'm not sure if it's guaranteed that way, but I liked that that data type was simpler, and just by excluding the ones with operations we could ensure getting only one message per instance.